_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -17,22 +17,23 @@ acknowledgments:
   authors:
     - Brian Bors
     - Carlos Duarte
+  previous_authors:
     - Wilco Fiers
 ---
 
 ## Applicability
 
-This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+This rule applies to each [link history state][] of any [semantic link][], for which all the following is true:
 
-- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
-- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
-- **different hue**: the element's [foreground colors][foreground color] and the [foreground color][] of the **non-link line text** elements have a [different hue][] in at least one [link history state][], or the element's [background colors][background color] and the [background color][] of the **non-link line text** elements have a [different hue][] in at least one [link history state][].
+- **link text**: the [semantic link][] element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the [semantic link][] element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
+- **different hue**: the [foreground colors][foreground color] of the [semantic link's][semantic link] **link text** and the [foreground color][] of the **non-link line text** elements have a [different hue][], or the [semantic link's][semantic link] [background colors][background color] and the [background color][] of the **non-link line text** elements have a [different hue][].
 
 ## Expectation
 
-For each [link history state][] with a **different hue** there exists at least one [visible][] [inclusive descendant][] and one [adequate inspection state][] of the test target such that at least one of the following is true:
+For each [link history state][] of the [semantic link][] there exists at least one [adequate inspection state][] and at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that at least one of the following is true:
```

Comment:
The "For each link history state" part doesn't make sense with the new Applicability.
I'm struggling a bit to find a nice formulation…
"For each test target, there exist at least one adequate inspection state of the link with this history state" just sounds bad.

Reply:
I don't agree that it sounds bad :-)
I'm changing it accordingly.